client/extensions/woocommerce/app/order/order-create/customer-card.js

```diff
@@ -0,0 +1,245 @@
+/** @format */
```

Comment:
Yeah, it marks that prettier's been run on these files. I think it's meant to be left in, so that we can re-run prettier on changes in the pre-commit hook.